homeassistant/components/sensor/glances.py

```diff
@@ -40,7 +40,8 @@
     'process_running': ['Running', 'Count'],
     'process_total': ['Total', 'Count'],
     'process_thread': ['Thread', 'Count'],
-    'process_sleeping': ['Sleeping', 'Count']
+    'process_sleeping': ['Sleeping', 'Count'],
+    'cpu_temp': ['CPU Temp', '°'],
```

Comment:
I was unsure the driver provides Celsius everywhere, maybe it provides Farenheit if the locale is less International System friendly ;D.

How should I implement? Should I get user preferences from somewhere? Glances doesn't say what the value units are, so I cannot convert automatically I think.